README.md

```diff
@@ -2,7 +2,135 @@
+
+| sample  | fastq_1                    | fastq_2                    | assembly                     |
+| ------- | -------------------------- | -------------------------- | ---------------------------- |
+| SAMPLE1 | /path/to/sample1_fastq1.fq | /path/to/sample1_fastq2.fq | /path/to/sample1_assembly.fa |
+| SAMPLE2 | /path/to/sample2_fastq1.fq |                            |                              |
+
+The columns are defined as follows:
+
+- `sample`: The unique sample identifier to associate with the reads (and optionally the assembly).
+- `fastq_1`: A URI (ex: a file path or web address) to either single-end FASTQ-formatted reads or one pair of pair-end FASTQ-formatted reads.
+- `fastq_2`: (Optional) If `fastq_1` is paired-end, then this field is a URI to reads that are the other pair of reads associated with `fastq_1`.
+- `assembly`: (Optional) A URI to an assembly associated with the sample, so that it may be referenced on the command line by the sample identifier for use as the reference for the whole pipeline. However, it may be easier to leave these fields blank and specify the reference using the `--refgenome` parameter.
+
+The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Please see [assets/samplesheet.csv](assets/samplesheet.csv) to see an example of a samplesheet for this pipeline.
+
+# Parameters
+
+## Mandatory
+
+The mandatory parameters are as follows:
+
+- `--input`: a URI to the samplesheet
+- `--output`: the directory for pipeline output
+
+## Optional
+
+The optional parameters are as follows:
+
+### Reference
+
+- `--refgenome`: a URI to the reference genome to use during pipeline analysis
+- `--reference_sample_id`: the sample identifier of a sample in the samplesheet that contains a provided `assembly` to use as a reference genome during pipeline analysis
+
+Please use only one of `--refgenome` or `--reference_sample_id` and not both.
+
+### SNVPhyl Parameters
+
+- `--window_size`: The window size for determining whether a region is high density.
+- `--density_threshold`: The minimum number of SNVs within the window size for a region to be considered high density.
+- `--min_coverage_depth`: The minimum depth of coverage for a position within the genome to pass the mapping quality check and be included in the analysis.
+- `--min_mapping_percent_cov`: The total percentage of positions within the genome that must have a depth of coverage greater than the minimum depth of coverage specified in order to pass the mapping quality check and be included in the analysis.
+- `--min_mean_mapping_quality`: The minimum mean mapping quality score for all reads in a pileup to be included in the analysis.
+- `--snv_abundance_ratio`: The proportion of reads required to support a variant to be included in the analysis.
+- `--min_repeat_length`: The minimum length when identifying repeats.
```

Comment:
Change to *identifying repeats __on the reference genome__*.

Reply:
Updated in 2be4bd2038d29cb4e55e43ed46e76c0d65f6b9b4